src/DataDepsGenerators.jl

```diff
@@ -46,41 +80,51 @@ include("Figshare.jl")
 include("JSONLD/JSONLD.jl")
 
 function message(meta)
-    escape_multiline_string("""
-    Dataset: $(meta.fullname)
-    Website: $(meta.website)
-    $(meta.description)
-    """) |> strip
+    netString =  """
+    register(DataDep(
+        \"$(meta.shortname)\",
+        \"\"\""""
+    netString = format_meta(meta.website, netString, "Website")
+    netString = format_meta(format_authors(meta.author), netString, "Author", true)
+    netString = format_meta(meta.maintainer, netString, "Maintainer", true)
+    netString = format_meta(meta.license, netString, "License", true)
+    netString = format_meta(meta.publishedDate, netString, "Date of Publication", true)
+    netString = format_meta(meta.createDate, netString, "Date of Creation", true)
+    netString = format_meta(meta.modifiedDate, netString, "Date of Modification", true)
+    netString = format_meta(meta.description, netString, missing,  true)
+    netString = format_meta(meta.paperCite, netString, "Please cite this paper", true)
+    netString = format_meta(meta.datasetCite, netString, "Please cite this dataset", true)
+    netString = strip(netString * "\n\"\"\",")
+    netString = format_meta(meta.dataurls, netString)
+    netString = netString * ","
+    netString = format_meta(format_checksums(meta.datachecksums), netString)
+    netString = strip(netString * "\n))")
+
+    netString
 end
 
-
-function data_shortnamename(repo::DataRepo, meta)
-    short_name = meta.fullname
-    reduce((s,r)->replace(s, r, ""), short_name, ['\\', '/', ':', '*', '?', '<', '>', '|'])
+function format_meta(data::Any, netString, label=missing, indentbool=false)
+    if ismissing(data)
+        return netString
+    else
+        new_data = (indentbool? indent(string(data)) : string(data))
+        if ismissing(label)
+            return netString * "\n" * new_data
+        else
+            return netString * "\n" * label * ": " * new_data
+        end
+    end
```

Comment:
I suggest instead:

Make `indentbool` a kwarg called `indent_field`,
and default it to `true` since that is what you mostly have it set to.

and I suggest the default value for `label`  should be `""`  because it isn't a missing label, it is an empty label.

and use dispatch to handle the missings.

So it would look something like this:

```
format_meta(::Missing, args...; kwargs...) = ""

function format_meta(data::Any, label=""; indent_field=true)
     if label != ""
         label*=":"
     end
     field = (indent_field ? indent(string(data)) : string(data))
     return "\n" * label * field
end
```

Reply:
and for example

```
netString = format_meta(meta.description, netString, missing,  true)
```

would become

```
netString *= format_meta(meta.description)
```

Reply:
In general I'm not too sure on what `indentbool` is for,

Maybe it can be simplified to not having it at all.

eg.
```
format_meta(::Missing, args...; kwargs...) = ""

function format_meta(data::Any, label="")
     if label != ""
         label*=":\t"
     end
     field = string(data)
     return "\n" * label * field
end
```

Would do the same thing, I think.

I do like the idea of putting a tab between fieldnames and there values.
It seems semantic